.github/workflows/test.yaml

```
@@ -134,6 +134,12 @@ jobs:
           # Just Python 3.12 with default settings.
           - python-version: '3.12'
 
+          # Newest macOS and oldest Python (major) supported versions.
+          - python-version: '3.8.0'
```

3.8.0 has a different failue - https://github.com/twisted/twisted/actions/runs/6131008336/job/16640617797?pr=11974#step:10:13916

try something newer

```suggestion
          - python-version: '3.8.10'
```